src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Orchestrators/PledgeOrchestratorTests.cs

```diff
@@ -162,6 +162,52 @@ public async Task GetApplications_Returns_Valid_ViewModel()
             });
         }
 
+        [Test]
+        public async Task GetApplications_With_Status_Pending_Renders_Continue_Button()
```

Comment:
I'd be tempted to fold these two tests into a single test with multiple test cases - you could pass in parameters for:
a) how many pending applications there are
b) the expected "renderbutton" boolean

You could then use test cases for 0,1 and 2 pending applications.

And then maybe add an additional test to prove that applications of other statuses do not result in the button being rendered

Reply:
Created two tests and added multiple test cases to test the number of pending applications and behaviors when non-pending applications are passed.